backend/src/openbeheer/zaaktype/tests/test_zaaktype_template_endpoint.py

```diff
@@ -46,3 +46,5 @@ def test_retrieve_list(self):
                 "Vertrouwelijkheidaanduiding: openbaar",
             ],
         )
+        # Not required field, which cannot be null. So it needs to be undefined.
+        self.assertNotIn("verantwoordingsrelatie", basis["waarden"])
```

Comment:
Yeah, tried finishing what I stared in 6909572f   
in https://github.com/maykinmedia/open-beheer/pull/308/files#diff-009de331e84d9d97d0a92bfeb94e59723234c159ce8cb97ec87bac879a761731